### PR TITLE
style: fix dropdown overflow issue on drop page

### LINKED ISSUE #9766: Dropdown overflow on drop page
minor, but there should probably be some more space under the drop down
image

### PR DESCRIPTION
**Pull request by floyd-li**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #9766
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/koda_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

CleanShot 2024-04-11 at 11 48 58@2x

### CODE REVIEW

File: components/common/EmptyResult.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <section class="px-6 py-12 text-center">
+  <section class="px-6 py-12 mb-16 text-center">
```
Comment (hassnian) [inline components/common/EmptyResult.vue:2]: let's stay consistent and use padding

Comment (floyd-li) [reply components/common/EmptyResult.vue]: done

Comment (hassnian) [reply components/common/EmptyResult.vue]: actually here it's changing for every component that uses `ItemsGrid.vue`, 

you should add any extra margin inside `GenerativeLayout`

CleanShot 2024-04-11 at 11 26 55@2x

Comment (floyd-li) [reply components/common/EmptyResult.vue]: ohhh thanks for the tip. it's better this way :)

**Review by hassnian — APPROVED** [review]

**Review by daiagi — APPROVED** [review]

**Comment by prury** [comment]
pay 5 usd

**Comment by floyd-li** [comment]
> pay 5 usd

@prury seems the payment not succeed

**Comment by prury** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 8.26 USD/DOT ~ 1.211 $DOT
🧗 [13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A](https://polkadot.subscan.io/account/13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A)
🔗 [0xb8506d4cdc8b4cef41734e183bc591dec68528c68c7f590d1a7012889656589a](https://polkadot.subscan.io/extrinsic/0xb8506d4cdc8b4cef41734e183bc591dec68528c68c7f590d1a7012889656589a)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**